mods/cnc/mod.yaml

```diff
@@ -29,13 +18,22 @@ Packages:
 	temperat.mix
 	winter.mix
 	desert.mix
-	snow.mix
 	~movies-gdi.mix
 	~movies-nod.mix
 	~movies.mix
 	~scores.mix
 	~scores2.mix
 	~transit.mix
+	./mods/cnc/bits
+	snow.mix
```

Comment:
This will possibly break my improved snow palette (which is in bits).
We should consider extracting the content from snow.mix into a subfolder to avoid such issues.